doc/documentation.md

```diff
@@ -608,3 +610,37 @@ Response:
   ]
 }
 ```
+### Configuration setting for Authority Archive records expiration
+In order to provide an ability for a tenant to have specific retention period of authority archives, we need to add the below configuration in mod-settings.
+If no setting is provided by a tenant the retention period value would be taken from `AUTHORITY_ARCHIVES_EXPIRATION_PERIOD` environment variable
+
+#### Permissions
+To make a post call to mod-settings, user should have below permissions.
+```
+  mod-settings.entries.item.post
+  mod-settings.global.write.mod-entities-links
+```
+
+#### Example request
+```
+POST https://{okapi-location}/settings/entries
+  {
+    "id":"1e01066d-4bee-4cf7-926c-ba2c9c6c0001",
+    "scope": "authority-storage",
+    "key": "authority-archives-expiration",
+    "value":"{\"expirationEnabled\":true, \"retentionInDays\":10}"
+}
+```
+
+| parameter | Type        | Description                                                                                                           |
+|---------  |-------------|-----------------------------------------------------------------------------------------------------------------------|
+| `id`      | UUID        | id of type UUID should be provided.                                                                                   |
+| `scope`   | String      | Scope should be the module name. Here, it will be "authority-storage"                                                 |
+| `key`     | String      | Feature or Identifier name matching the setting we are enabling for. Here, it will be "authority-archives-expiration" |
+| `value`   | Json Object | Value object for this setting.                                                                                        |
+
+
+| Value options       | Type    | Description                                                                                                                                                                           |
+|---------------------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `expirationEnabled` | boolean | Indicates whether or not to enable expiration of Authority Archives for the tenant. Default value is false(disabled).                                                                 |
```

Comment:
Default is false?

Reply:
This was redundant, the implementation actually already handles it.